recipe/conda_forge_ci_setup/build_utils.py

```diff
@@ -135,17 +136,8 @@ def upload_package(feedstock_root, recipe_root, config_file):
                 print("Uploading to conda-forge with source channel '{}' is not allowed".format(source_channel))
                 return
 
-    upload_script_path = os.path.join(recipe_root, 'conda_forge_ci_setup', 'upload_or_check_non_existence.py')
-    if not os.path.exists(upload_script_path):
-        upload_script_path = os.path.join(os.path.dirname(__file__), 'upload_or_check_non_existence.py')
-
-    import importlib.util
-    spec = importlib.util.spec_from_file_location('upload_or_check_non_existence', upload_script_path)
-    mod = importlib.util.module_from_spec(spec)
-    spec.loader.exec_module(mod)
-
```

Comment:
I removed this stuff since it is breaking the CI. We can add an option to smithy to specify a location from which to install a copy of the package.

Reply:
see here @isuruf

Reply:
If we remove this, how do we know that this package works?

Reply:
We won't know it works right now because none of the validation stuff is in smithy yet. The endpoints work, but we need changes to smithy first. So it doesn't matter for that. 

I can use the import magic instead of removing this section.